packages/create-gatsby/src/tracking.ts

```diff
@@ -0,0 +1,46 @@
+import fetch from "node-fetch"
+import { v4 as uuidv4 } from "uuid"
+import { getConfigStore } from "./get-config-store"
+
+const store = getConfigStore()
+const gatsbyCliVersion = require(`../package.json`).version
+const analyticsApi =
+  process.env.GATSBY_TELEMETRY_API || `https://analytics.gatsbyjs.com/events`
+
+let machineId = store.get(`telemetry.machineId`)
```

Comment:
We might need to move this to be after the function
```suggestion
let machineId = getMachineId()
```